env-defaults

```diff
@@ -180,11 +180,8 @@ WORK_REAPER_BATCH_SIZE=2000
 # The time (in seconds) between invocations of the work failer service
 WORK_FAILER_PERIOD_SEC=120
 
-# The time (in seconds) the work-item update queue processor will wait before resuming after an error
-# Used to avoid a tight loop
-WORK_ITEM_UPDATE_QUEUE_PROCESSOR_DELAY_AFTER_ERROR_SEC=1
-
-WORK_ITEM_UPDATE_QUEUE_PROCESSOR_COUNT=3
+# The number of update queue processors to run for both large and small work-item queues
+WORK_ITEM_UPDATE_QUEUE_PROCESSOR_COUNT=1
```

Comment:
For deployments this is set using `tf_k8s_service_pod_replicas`, which is also used for the work scheduler service. I debated about adding a separate config just for the work updater, but decided not to until we think they need to be different (to avoid yet another var). Currently this is set to 3 (at least for UAT). If you want me to add a separate var to control this I can.

Reply:
Yeah I commented in the ci-cd PR - definitely want to control them with separate configuration values.